openlibrary/plugins/openlibrary/home.py

```diff
@@ -151,15 +152,15 @@ def format_list_editions(key):
     """
     if 'env' not in web.ctx:
         delegate.fakeload()
-    
+
     list = web.ctx.site.get(key)
```

Comment:
Not a critique of this CR, but we should not be overriding the list function. Consider `lst` instead or `seed_list`.